Sources/Sentry/include/SentryMsgPackSerializer.h

```diff
@@ -0,0 +1,33 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol SentryStreamable
```

Comment:
`m`: We could maybe have written that class in Swift. I think we should make it a habit to write new classes without dependencies on the rest of the SDK in Swift.

Reply:
> write new classes without dependencies 

Even those with deps? Are we free now to write just swift unles we need swizzling?

Reply:
@brustolin and I had a chat yesterday. @brustolin is working on making it possible to have ObjC and Swift code mixed in our codebase. If we succeed with the setup then yes 😃 💪

Reply:
We can refactor this in swift in another PR